config/cylc_hpc/flow.cylc

```diff
@@ -42,40 +49,99 @@
             mkdir -p $tracker_dir
             mkdir -p $h5_dir
         """
+        platform = oscar
+        execution time limit = PT1H
+        [[[directives]]]
+            --mem = 4G
+            --cpus-per-task = 6
```

Comment:
You probably don't want to have more than one CPU for this – you're just making directories. You might not even need to dispatch via slurm – the scheduler should be able to handle this task.